src/devices/Ina226/DelayHelper.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Device
     /// <summary>
     /// Helpers for short waits.
     /// </summary>
-    internal static class DelayHelper
+    internal static class Ina226DelayHelper
```

Comment:
I have an idea on how to properly fix the problem here, but not sure when I get to it.

Reply:
I just went ahead and moved it fully into the iot.device.ina226 namespace in hopes that would resolve the dependency issue.

Reply:
PR #1759 should properly solve it.